### DISCUSSION TITLE
entitlement file option/flag

### DISCUSSION THREAD

**Discussion by lastlink** [fastlane/fastlane#19210] [Ideas] [answered]
* option to pass in entitlement file when notarizing
* example https://www.xamboy.com/2020/05/28/net-core-application-macos-packaging-and-notarization/
* https://gist.github.com/rdelrosario/9e0f4e79a468082424b5bd4714c34fb0

**Comment by macblazer** [comment, marked as answer]
Entitlements should not be added to the `notarization` action because entitlements are part of codesigning and not part of notarization.  The gist you linked to calls `codesign` using the entitlements, but does not use them for the actual notarization within that script.

Fastlane separates codesigning from notarization because usually codesigning is done as part of the build process.  https://docs.fastlane.tools/actions/#code-signing shows the many different actions that specifically involve codesigning.  Many of the _Building_ actions also do codesigning as part of the build.
